dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/message/EmailMessageSender.java

```diff
@@ -123,6 +123,7 @@ public OutboundMessageResponse sendMessage( String subject, String text, String
         if ( emailConfig.getHostName() == null )
         {
             status.setOk( false );
+            status.setDescription( "Email host configuration not found" );
```

Comment:
This looks fine to me. 
Just a suggestion, we have EmailResponse Enum which has responseMessage string for each element. This responseMessage string can also be used instead of a description. We can add more specific Enum elements, for example, HOST_CONFIG_ERROR( "Host configuration not found" ).

Reply:
Yes. I agree. I saw a lot of inconsistent usages in the codebase. There are plenty of other classes using the logic of `OutboundMessageResponse` and/or `sendMessage()`

I didn't touch those, because in some cases I didn't understand completely what the message should be, etc.

But yes, if this part of the codebase is still in use, then there are possibilities for improvements and refactoring to make code cleaner, consistent and more readable.

Reply:
I think it makes sense to reuse that enum yeah. Good suggestion @zubaira